README.md

```diff
@@ -40,6 +40,23 @@ The entry in the application's `/server/datasources.json` will look like this:
 
 Edit `datasources.json` to add any other additional properties that you require.
 
+If your username or password contains special characters like `@`, `$` etc, encode the whole
+username or password using [percent-encoding](https://tools.ietf.org/html/rfc3986#section-2.1).
+
+Eg: `a!$@xyz` would become `%61%21%24%40%78%79%7A`.
+
+Here is a function to help you generate percent-encoded string:
+
+```js
+function percentEncode(string) {
```

Comment:
No need to write a custom encoder, just use the built-in [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent).